### PR TITLE
Fix Behat

### PR DESCRIPTION
**Pull request by meyerbaptiste**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | N/A
| License       | MIT
| Doc PR        | N/A

See https://github.com/willdurand/Negotiation/pull/92#issuecomment-299248503

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "symfony/property-access": "^2.7 || ^3.0",
         "symfony/property-info": "^3.1",
         "symfony/serializer": "^3.1",
-        "willdurand/negotiation": "^2.0.3"
+        "willdurand/negotiation": "~2.2.1"
```
Comment (soyuka) [inline composer.json]: `^2.0.3 <2.3`? 

https://github.com/api-platform/core/pull/1099#discussion_r114942093

Comment (meyerbaptiste) [inline composer.json]: Done!

**Comment by dunglas** [comment]
Thanks @meyerbaptiste

**Comment by meyerbaptiste** [comment]
Don't forget to merge it in the master branch too @dunglas!

**Comment by dunglas** [comment]
Done.

**Pull request merged**